lib/worksheet.js

```diff
@@ -236,7 +236,7 @@ XlsxStreamReaderWorkSheet.prototype._handleWorkSheetNode = function (nodeData) {
             if (self.workBook.hasFormatCodes) {
               var formatId = self.workBook.xfs[workingCell.attributes.s].attributes.numFmtId
               var format = self.workBook.formatCodes[formatId]
```

Comment:
if we have to use a predefined format, this variable will turn out undefined

Reply:
Hm, maybe binary plus like "+formatId" will work faster? I am not sure

Reply:
Yes, maybe it is ... I didn't think about that.

Reply:
https://jsperf.com/number-vs-plus-vs-toint-vs-tofloat/39 

looks like that bitwise not is faster?